chapters/synchronous.tex

```diff
@@ -840,7 +840,7 @@ \subsection{Sub-Clock Conversion Operators}\label{sub-clock-conversion-operators
 
 \section{Clocked When-Clause}\label{clocked-when-clause}
 
-In addition to the previously discussed conditional \lstinline!when!-clause, a \emph{clocked} \lstinline!when!-clause\index{clocked!when-clause@\robustinline{when}-clause}\index{when-clause@\robustinline{when}-clause!clocked} is introduced:
+In addition to the previously discussed conditional \lstinline!when!-clause (see \cref{when-equations}), a \emph{clocked} \lstinline!when!-clause\index{clocked!when-clause@\robustinline{when}-clause}\index{when-clause@\robustinline{when}-clause!clocked} is introduced:
```

Comment:
Since there are no clocked when-statements, isn't it better to just compare with `when`-equation, as this is also where we're adding the cross reference in the other direction?
```suggestion
In addition to the previously discussed \lstinline!when!-equation (see \cref{when-equations}), a \emph{clocked} \lstinline!when!-clause\index{clocked!when-clause@\robustinline{when}-clause}\index{when-clause@\robustinline{when}-clause!clocked} is introduced:
```

By the way, the _is introduced_ is the kind of formulation that makes the synchronous features not feel like first class citizens of the language.

Reply:
Yes. But that seems like a larger change that should be coordinated.

Reply:
Sure.